package.json

```diff
@@ -62,11 +63,7 @@
     "webpack": "5.87.0",
     "webpack-dev-server": "3.11.3"
   },
-  "workspaces": [
-    "charts/*",
-    "chart-modules/*"
-  ],
   "resolutions": {
-    "**/caniuse-lite": "1.0.30001504"
+    "caniuse-lite": "1.0.30001504"
```

Comment:
Don't think `resolutions` works for pnpm. You want to use `{ "pnpm": { "overrides": ... } }`

Reply:
It does. I left it like this to keep the change to a minumum but might be better to follow what pnpm suggets